### ISSUE TITLE
arabic recognition model gives bad results for arabic numbers

### ISSUE THREAD

**Issue by amroghoneim** [PaddlePaddle/PaddleOCR#8180]
I am going through the tutorial and testing out some images that include only arabic numbers.

I usually see bad results even though its quite accurate for text.

Can this be resolved using some parameters? or do I need to finetune the recognition model ?

**Comment by mohamedbakhet** [comment]
I face same problem

**Comment by amroghoneim** [comment]
@MissPenguin Please advise.

**Comment by LDOUBLEV** [comment]
Try finetune training. (The data we have is really limited.)

If it is a bad case, it is recommended to send some samples to analyze.

**Comment by tink2123** [comment]
@amroghoneim  Could you provide prediction commands and badcase?  Let's reproduce the effect.

**Comment by amroghoneim** [comment]
Sure @tink2123 

So this is a sample code

```
from paddleocr import PaddleOCR,draw_ocr


ocr = PaddleOCR( lang='ar', max_text_length = 50, rec_batch_num = 1) # need to run only once to download and load model into memory
result = ocr.ocr(np.array(img), cls=False, det = False)
```

example images with numbers only:
![image](https://user-images.githubusercontent.com/37261960/204888908-d5577c47-d192-422a-be6f-630ee0c70cd2.png)

![image](https://user-images.githubusercontent.com/37261960/204888989-d435f509-09cf-47a6-8f25-05fdef866d58.png)


![image](https://user-images.githubusercontent.com/37261960/204889206-3f6ae21c-1b7d-4978-bb91-62c25f88e04f.png)


By the way I also see degradation when combining numbers with text, and when doing recognition on multiple words compared to splitting them each one on their own.

**Comment by Topdu** [comment]
Thanks!
This problem is caused by incorrectly processing the Arabic number labels during training, and we will subsequently update the model after retraining.
You can also perform finetune training on the Arabic numbers dataset to solve this problem.

**Comment by amroghoneim** [comment]
@Topdu will this be released anytime soon?

Also the finetuning process seems to be hard for me to get my head around when following the documentation.
May I ask here what are the steps I need to follow exactly? how should the dataset look like (example dataset url is not working)? is there a training script I can use off the bat? What does the GT label file look like and so on.

**Comment by amroghoneim** [comment]
@Topdu Any comments please?

**Comment by Topdu** [comment]
As soon as possible.

For training documents, please refer to [rec training doc](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/recognition_en.md).

**Comment by AstonyJ** [comment]
遇到了同样的问题，使用latin_PP-OCRv3_rec模型，想问下是否有更新之后的模型呢？

**Comment by vk1996** [comment]
The arabic numbers prediction of SVTR arabic model  is poor . any updates on this please ?

**Comment by Topdu** [comment]
[PP-OCRv4 arabic model](https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/paddleocr.py#L135C27-L135C27)

**Comment by vk1996** [comment]
is the config file same as v3 ?

**Comment by Hegelim** [comment]
> [PP-OCRv4 arabic model](https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/paddleocr.py#L135C27-L135C27)

This is an inference model, what about pre-trained model for Arabic?

**Comment by Hegelim** [comment]
Is the dictionary same?
